{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "moderate"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "Security update for python-Pillow",
				"title": "Title of the patch"
			},
			{
				"category": "description",
				"text": "This update for python-Pillow fixes the following issues:\n\n  * CVE-2016-9190: Pillow allows context-dependent attackers to execute arbitrary code by using the \n    \\'crafted image file\\' approach, related to an \\'Insecure Sign Extension\\' issue affecting the \n    ImagingNew in Storage.c component. (bsc#1008846)\n\n  * CVE-2016-3076: Heap-based buffer overflow in the j2k_encode_entry function allows remote attackers \n    to cause a denial of service (memory corruption) via a crafted Jpeg2000 file. (bsc#973786) \n\n",
				"title": "Description of the patch"
			},
			{
				"category": "details",
				"text": "SUSE-OpenStack-Cloud-7-2018-816,SUSE-Storage-4-2018-816",
				"title": "Patchnames"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "SUSE ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "self",
				"summary": "URL of this CSAF notice",
				"url": "https://ftp.suse.com/pub/projects/security/csaf/suse-su-2018_1174-1.json"
			},
			{
				"category": "self",
				"summary": "URL for SUSE-SU-2018:1174-1",
				"url": "https://www.suse.com/support/update/announcement/2018/suse-su-20181174-1/"
			},
			{
				"category": "self",
				"summary": "E-Mail link for SUSE-SU-2018:1174-1",
				"url": "https://lists.suse.com/pipermail/sle-security-updates/2018-May/003980.html"
			},
			{
				"category": "self",
				"summary": "SUSE Bug 1008846",
				"url": "https://bugzilla.suse.com/1008846"
			},
			{
				"category": "self",
				"summary": "SUSE Bug 973786",
				"url": "https://bugzilla.suse.com/973786"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2016-3076 page",
				"url": "https://www.suse.com/security/cve/CVE-2016-3076/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2016-9190 page",
				"url": "https://www.suse.com/security/cve/CVE-2016-9190/"
			}
		],
		"title": "Security update for python-Pillow",
		"tracking": {
			"current_release_date": "2018-05-08T20:14:49Z",
			"generator": {
				"date": "2018-05-08T20:14:49Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf.pl",
					"version": "1"
				}
			},
			"id": "SUSE-SU-2018:1174-1",
			"initial_release_date": "2018-05-08T20:14:49Z",
			"revision_history": [
				{
					"date": "2018-05-08T20:14:49Z",
					"number": "1",
					"summary": "Current version"
				}
			],
			"status": "final",
			"version": "1"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-Pillow-2.8.1-4.3.2.aarch64",
								"product": {
									"name": "python-Pillow-2.8.1-4.3.2.aarch64",
									"product_id": "python-Pillow-2.8.1-4.3.2.aarch64"
								}
							}
						],
						"category": "architecture",
						"name": "aarch64"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-Pillow-2.8.1-4.3.2.s390x",
								"product": {
									"name": "python-Pillow-2.8.1-4.3.2.s390x",
									"product_id": "python-Pillow-2.8.1-4.3.2.s390x"
								}
							}
						],
						"category": "architecture",
						"name": "s390x"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-Pillow-2.8.1-4.3.2.x86_64",
								"product": {
									"name": "python-Pillow-2.8.1-4.3.2.x86_64",
									"product_id": "python-Pillow-2.8.1-4.3.2.x86_64"
								}
							}
						],
						"category": "architecture",
						"name": "x86_64"
					},
					{
						"branches": [
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud 7",
								"product": {
									"name": "SUSE OpenStack Cloud 7",
									"product_id": "SUSE OpenStack Cloud 7",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-openstack-cloud:7"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE Enterprise Storage 4",
								"product": {
									"name": "SUSE Enterprise Storage 4",
									"product_id": "SUSE Enterprise Storage 4",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:ses:4"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Pillow-2.8.1-4.3.2.aarch64 as component of SUSE OpenStack Cloud 7",
					"product_id": "SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.aarch64"
				},
				"product_reference": "python-Pillow-2.8.1-4.3.2.aarch64",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Pillow-2.8.1-4.3.2.s390x as component of SUSE OpenStack Cloud 7",
					"product_id": "SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.s390x"
				},
				"product_reference": "python-Pillow-2.8.1-4.3.2.s390x",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Pillow-2.8.1-4.3.2.x86_64 as component of SUSE OpenStack Cloud 7",
					"product_id": "SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.x86_64"
				},
				"product_reference": "python-Pillow-2.8.1-4.3.2.x86_64",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Pillow-2.8.1-4.3.2.aarch64 as component of SUSE Enterprise Storage 4",
					"product_id": "SUSE Enterprise Storage 4:python-Pillow-2.8.1-4.3.2.aarch64"
				},
				"product_reference": "python-Pillow-2.8.1-4.3.2.aarch64",
				"relates_to_product_reference": "SUSE Enterprise Storage 4"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Pillow-2.8.1-4.3.2.x86_64 as component of SUSE Enterprise Storage 4",
					"product_id": "SUSE Enterprise Storage 4:python-Pillow-2.8.1-4.3.2.x86_64"
				},
				"product_reference": "python-Pillow-2.8.1-4.3.2.x86_64",
				"relates_to_product_reference": "SUSE Enterprise Storage 4"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2016-3076",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2016-3076"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "Heap-based buffer overflow in the j2k_encode_entry function in Pillow 2.5.0 through 3.1.1 allows remote attackers to cause a denial of service (memory corruption) via a crafted Jpeg2000 file.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"fixed": [
					"SUSE Enterprise Storage 4:python-Pillow-2.8.1-4.3.2.aarch64",
					"SUSE Enterprise Storage 4:python-Pillow-2.8.1-4.3.2.x86_64",
					"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.aarch64",
					"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.s390x",
					"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2016-3076",
					"url": "https://www.suse.com/security/cve/CVE-2016-3076"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 973786 for CVE-2016-3076",
					"url": "https://bugzilla.suse.com/973786"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"SUSE Enterprise Storage 4:python-Pillow-2.8.1-4.3.2.aarch64",
						"SUSE Enterprise Storage 4:python-Pillow-2.8.1-4.3.2.x86_64",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.aarch64",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.s390x",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 5.5,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H",
						"version": "3.0"
					},
					"products": [
						"SUSE Enterprise Storage 4:python-Pillow-2.8.1-4.3.2.aarch64",
						"SUSE Enterprise Storage 4:python-Pillow-2.8.1-4.3.2.x86_64",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.aarch64",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.s390x",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2018-05-08T20:14:49Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2016-3076"
		},
		{
			"cve": "CVE-2016-9190",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2016-9190"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "Pillow before 3.3.2 allows context-dependent attackers to execute arbitrary code by using the \"crafted image file\" approach, related to an \"Insecure Sign Extension\" issue affecting the ImagingNew in Storage.c component.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"fixed": [
					"SUSE Enterprise Storage 4:python-Pillow-2.8.1-4.3.2.aarch64",
					"SUSE Enterprise Storage 4:python-Pillow-2.8.1-4.3.2.x86_64",
					"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.aarch64",
					"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.s390x",
					"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2016-9190",
					"url": "https://www.suse.com/security/cve/CVE-2016-9190"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1008846 for CVE-2016-9190",
					"url": "https://bugzilla.suse.com/1008846"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 973786 for CVE-2016-9190",
					"url": "https://bugzilla.suse.com/973786"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"SUSE Enterprise Storage 4:python-Pillow-2.8.1-4.3.2.aarch64",
						"SUSE Enterprise Storage 4:python-Pillow-2.8.1-4.3.2.x86_64",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.aarch64",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.s390x",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.8,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
						"version": "3.0"
					},
					"products": [
						"SUSE Enterprise Storage 4:python-Pillow-2.8.1-4.3.2.aarch64",
						"SUSE Enterprise Storage 4:python-Pillow-2.8.1-4.3.2.x86_64",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.aarch64",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.s390x",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.3.2.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2018-05-08T20:14:49Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2016-9190"
		}
	]
}
